
package org.ejbca.core.protocol.ws;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.1.3
 * 2018-09-27T08:23:56.026+02:00
 * Generated source version: 3.1.3
 */

@WebFault(name = "UserDataSourceException", targetNamespace = "http://ws.protocol.core.ejbca.org/")
public class UserDataSourceException_Exception extends Exception {
    
    private org.ejbca.core.protocol.ws.UserDataSourceException userDataSourceException;

    public UserDataSourceException_Exception() {
        super();
    }
    
    public UserDataSourceException_Exception(String message) {
        super(message);
    }
    
    public UserDataSourceException_Exception(String message, Throwable cause) {
        super(message, cause);
    }

    public UserDataSourceException_Exception(String message, org.ejbca.core.protocol.ws.UserDataSourceException userDataSourceException) {
        super(message);
        this.userDataSourceException = userDataSourceException;
    }

    public UserDataSourceException_Exception(String message, org.ejbca.core.protocol.ws.UserDataSourceException userDataSourceException, Throwable cause) {
        super(message, cause);
        this.userDataSourceException = userDataSourceException;
    }

    public org.ejbca.core.protocol.ws.UserDataSourceException getFaultInfo() {
        return this.userDataSourceException;
    }
}
